Microsoft.AVS.VMFS/Microsoft.AVS.VMFS.psm1

```diff
@@ -1083,3 +1092,77 @@ function Mount-VmfsDatastore {
     }
                
   }
+
+
+<#
+    .SYNOPSIS
+     This function list all VMFS datastores accessible to host(s) under the given ESXi Cluster.
+
+     1. vSphere Cluster Name
```

Comment:
?

Reply:
We want to collect all existing datastores accessible to esxi hosts under the cluster name.
 OR you mean i should remove  **1. vSphere Cluster Name**

Reply:
Not needed. I'll remove.

Reply:
fixed.